src/components/data/listing/TableView.js

```diff
@@ -87,7 +87,12 @@ function TableView(props) {
     const tableColumns = getTableColumns(isMedium, isLarge);
 
     return (
-        <TableContainer component={Paper}>
+        <TableContainer
+            component={Paper}
+            style={{
+                height: "60vh",
+            }}
+        >
```

Comment:
I can adjust the height once we have all the elements in the page.

Reply:
Hmm Each page might have different set of components 🤔. Not sure how it work in different screens. But we can try I guess 👍